Verified ContentService documentation for v8

poornimanayar opened the pull request:
This is for issue https://github.com/umbraco/UmbracoDocs/issues/2128. I have verified `ContentService` docs for v8. The `ConsentService` has separate docs for v7 and v8. But I have deviated from it as the `ContentService` only needed updated API docs link to v8. I can always make it consistent with `ConsentService` and have separate v7 and v8 pages.

sofietoft commented:
I think it would be great to keep some consistency here 😄 

Would you mind splitting it up into two articles instead? 
And also, I noticed that the link for Umbraco 8 API isn't correct? 🤔 - or it currently gives a 404..
For `ConsentService`, it looks like `IConsentService` is referenced instead on the Umbraco 8 version?

Now, I'm no expert in this at all 😅 But would it be possible that we should link to `IContentService` on this one as well?

poornimanayar commented:
@sofietoft , That was me being careless! I have fixed it all now and I have split the v7 and v8 into two files. I wasn't sure about removing the mention of v6 in there, so I have retained it for now.

sofietoft commented:
Thanks @poornimanayar ! 😄 

I think we should get rid of the mention of Umbraco 6 .. on both articles actually!
Doesn't seem relevant anymore 😁

poornimanayar commented:
Thats done now! Now I have a direction for other services. I shall try and do more during this week if I can :-)

sofietoft commented:
Sounds great @poornimanayar ! 😄 

I'll run this past the team as well, as I'm no expert in this field 😅

poornimanayar commented:
Yes please :-)

sofietoft commented:
I've had a chat with Marc about this now, and we have a few notes for you.

The whole "Getting the Service" part needs to be updated for Umbraco 8 as well. Seems that there are some difference on how to do that, between Umbraco 7 and 8 🤔

So, the whole idea with documenting the Services here on Github, is to link off to the auto-generated docs on the service, a few notes on what it's used for, a few examples of getting a reference to it, and finally a small and very generic use-sample 😁

sofietoft commented on Reference/Management/Services/ContentService/Index-v7.md:
```diff
@@ -0,0 +1,46 @@
+---
+keywords: services content service
+versionFrom: 7.0.0
+versionRemoved: 8.0.0
```
@dampee - By adding the `versionRemoved`, isn't it going to say that the ContentService has been removed in Umbraco 8? It hasn't been removed, has it? 🤔

dampee replied:
@sofietoft Let me verify that. :-o

dampee replied:
It will mention 7.0.0 - 8.0.0 instead of 7.0.0+
So I think the change is ok.

sofietoft commented:
@poornimanayar - quick update here!

We (the Documentation Curators) have talked about this whole deal about getting all the articles about services up to date for Umbraco 8.
So, if you have a look at the MediaService and how that's been updated for Umbraco 8: https://our.umbraco.com/Documentation/Reference/Management/Services/MediaService

We're looking for a structure similar to that, containing:
* Paragraph explaining what they are
* Perhaps some tips on when to use them
* Link to API
* If relevant: "Real life" use sample
* Link to Services and Helpers documentation (https://our.umbraco.com/Documentation/Implementation/Services/)

How does that all sound to you? 😄

poornimanayar commented:
Sounds good @sofietoft an example will be super helpful and we have exactly that!

sofietoft commented:
Hi @poornimanayar - apologies for the late reply here!

I'll merge this PR now. Thanks for helping getting it updated 😄
